### PR TITLE
CRAN release v2.0.3

### PR DESCRIPTION
**Pull request by krlmlr**
## Action items

- Groom `NEWS.md` and `cran-comments.md`
- Wait for successful tests
- If needed, tweak and review PR
- Run `fledge::release()`
- Confirm CRAN e-mail
- When the package is accepted on CRAN, run `fledge::post_release()` (often handled by GitHub Actions)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,152 +1,59 @@
...
 
+## Features
 
-# igraph 2.0.2.9006
+- New `voronoi_cells()` to compute the Voronoi partitioning of a graph (#1173).
 
 ## Chore
 
+- Update revdepchecks.
+- Make libxml optional (#1282).
```
Comment (szhorvat) [inline NEWS.md:16]:
```suggestion
- Make libxml2 optional (#1282).
```

**Comment by szhorvat** [comment]
Not sure if this was submitted already. I recommend updating to the current latest commit from the C core (at the moment: https://github.com/igraph/igraph/commit/857a125069c226f266562b3781b82833fe1d59d9). I also suggest running revdepchecks, since there were lots of bugfixes in the C core recently, some of which fixed incorrect results.

**Comment by krlmlr** [comment]
I ran revdepchecks on igraph/igraph@5ba08607ec43f8a4fc9cc21933a1a21a274f5e02, do I need to run them again after updating sources?

**Comment by krlmlr** [comment]
Opening a new PR, incorporated the libxml2 change into main.

**Comment by szhorvat** [comment]
Yes, I would recommend running them. But to be honest, the chance of breakage is low, and if there is one, it's because the dependent package was doing something it shouldn't be doing.

**Pull request closed without merging**